apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -1088,6 +1088,18 @@ properties:
 
   - name: appBadge
     summary: Value of the badge for the application's springboard icon.
+    description: |
+        In order to change the app icon badge number you will need to register for notifications.
+        Normally this is done by using local notifications or a push service like FCM.
```

Comment:
FCM may not be a common term for non-Firebase users. Can you rephrase to `[Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) (FCM)`?

Reply:
:+1: